src/Components/Results/ProgramPage/ProgramPage.tsx

```diff
@@ -1,24 +1,115 @@
+  headingType: string;
+};
+
 const ProgramPage = ({ program }: ProgramPageProps) => {
-  const { uuid, programId } = useParams();
+  const IconRenderer: React.FC<IconRendererProps> = ({ headingType }) => {
+    const IconComponent = headingOptionsMappings[headingType];
+
+    if (!IconComponent) {
+      return null;
+    }
+
+    return <IconComponent />;
+  };
+
+  const { uuid } = useParams();
+
   return (
-    <>
-      <div>{program.name.default_message}</div>
-      <NavigatorCard navigators={program.navigators} />
-      <div>{program.name_abbreviated}</div>
-      <div>{program.estimated_value}</div>
-      <div>{program.category.default_message}</div>
-      <div>
-        <Link to={`/${uuid}/results/benefits/${programId}/navigators`}>Get help</Link>
+    <article className="program-page-container">
+      <section className="back-to-results-button-container">
+        <BackAndSaveButtons
+          handleTextfieldChange={() => {}}
+          navigateToLink={`/${uuid}/results/benefits`}
+          BackToThisPageText={<FormattedMessage id="results.back-to-results-btn" defaultMessage="BACK TO RESULTS" />}
+        />
+      </section>
+
+      <header className="program-header">
+        <div className="header-icon-box">
+          <div className="header-icon">
+            <IconRenderer headingType={program.category.default_message} />
+          </div>
+        </div>
+
+        <div className="header-text">
+          <p className="header-text-top">
+            <ResultsTranslate translation={program.category} />
+          </p>
+          <div className="divider"></div>
+          <h1 className="header-text-bottom">{program.name_abbreviated}</h1>
+        </div>
+      </header>
+
+      <section className="estimation">
+        <div className="estimation-text">
+          <div className="estimation-text-left">Estimated Annual Value</div>
+          <div className="estimation-text-right">${program.estimated_value}</div>
+        </div>
+        <div className="estimation-text">
+          <div className="estimation-text-left">Estimated Time to Apply</div>
+          <div className="estimation-text-right">
+            <ResultsTranslate translation={program.estimated_application_time} />
+          </div>
+        </div>
+      </section>
+
+      <div className="apply-online-button">
+        <Link to={program.apply_button_link.default_message} target="_blank" rel="noopener noreferrer">
+          Apply Online
+        </Link>
       </div>
-    </>
+
+      <section className="apply-box">
+        <h3>Get Help Applying</h3>
+        <ul className="apply-box-list">
+          {program.navigators.map((info, index) => (
+            <li key={index} className="apply-info">
+              <ResultsTranslate translation={info.name} />
+              <address className="address-info">
+                <p>Address info goes here</p> {/* replace with info.address */}
+                <a href={info.assistance_link.default_message}>
```

Comment:
`email`, `phone`, and `assistance_link` can be blank, if a navigator does not have one of those things. Could you make it so that if there is no default message, then the field does not show up.